scripts/build/Platform/Windows/build_config.json

```diff
@@ -172,6 +172,24 @@
       "TEST_RESULTS": "True"
     }
   },
+  "test_cpu_profile_vs2019": {
```

Comment:
two entries with the same name will make the json fail
this entry is also misleading since it is running the periodic suite
Is this WIP just to locally debug? if so, can you mark/comment?